runsc/sandbox/sandbox.go

```diff
@@ -1645,3 +1645,28 @@ func SetUserMappings(spec *specs.Spec, pid int) error {
 	}
 	return nil
 }
+
+// Mount mounts a filesystem in a container.
+func (s *Sandbox) Mount(cid, fstype, src, dest string) error {
```

Comment:
This seems custom-tailored to the EROFS case, since it takes in `src` as an image file which is specific to EROFS (as opposed to, say, mounting a goferfs that is backed by host files). I suggest renaming this to something like `MountImageFilesystem` or similar, to narrow down the scope of this function.

Reply:
Source being the image path seems to be correct. See "HOWTO Use EROFS" section in https://linuxreviews.org/EROFS. Example mount operation:`sudo mount -o loop myimage.erofs /mnt/tmp`. It is important that src is interpreted as image path only for EROFS. Other filesystems should interpret source in their own syntax. So I think this is general enough (mimics a general Mount operation).

Reply:
Ah but imageFile is opened unconditionally, I guess this is what Etienne is talking about.

Can you only open and donate imageFile FD if `fsType == erofs`.

Reply:
> Ah but imageFile is opened unconditionally, I guess this is what Etienne is talking about.
> 
> Can you only open and donate imageFile FD if `fsType == erofs`.

My bad, I should add a check on `fstype` in this function as well similar to what `containerManager.Mount` does. Will do. Thanks!